lib/Arguments/Signature.cpp

```diff
@@ -69,23 +69,6 @@ void Signature::dump() {
   }
 }
 
-std::string Signature::serialize() {
```

Comment:
Why did this get moved to the header?

Reply:
I encountered a linker error. Probably, I need to add a dependency to link `Signature::serialize()`, but I was not able to find what library should be listed.

Reply:
Should be: QSSCArguments
https://github.com/openqasm/qe-compiler/blob/f62b76dfdf5a839998f3047abb6d1be623608932/lib/Arguments/CMakeLists.txt#L13